docs/products/dragonfly/howto/migrate-ext-redis-df-console.rst

```diff
@@ -0,0 +1,116 @@
+Migrate external Redis®* databases to Aiven for Dragonfly®
+===========================================================================
+
+Redis is an in-memory data structure store commonly used as a database, cache, message broker, and queue. The Aiven Console provides an intuitive wizard to facilitate the migration of your external Redis databases to Aiven for Dragonfly.
+
+.. important::
+   The migration of databases from Google Cloud Memorystore for Redis is not supported at this time.
+
+Compatibility overview
+-----------------------
+Before migrating an external Redis database to Aiven for Dragonfly, carefully review your current Redis setup.
+
+* **Review database setup:** Examine your Redis database's data structures, storage patterns, and configurations.Identify any unique features, custom settings, and specific configurations.
+
+* **API compatibility:** While Dragonfly closely mirrors Redis API commands, there may be variations, especially with newer versions of Redis. For detailed insights on command compatibility, refer to the `Dragonfly API compatibility documentation <https://www.dragonflydb.io/docs/command-reference/compatibility>`_.
+
+
+
+Prerequisites
+-------------
+
+Before starting the migration process, ensure you have the following:
+
+* A target Aiven for Dragonfly service set up and ready. For setup instructions, see to :doc:`/docs/products/dragonfly/get-started`.
+* Source database information:
+
+  * **Hostname or connection string:** The public hostname, connection string, or IP address used to connect to the database, which should be :doc:`accessible from the public Internet </docs/platform/howto/public-access-in-vpc>`.
+  * **Port:** The port number used for connecting to the database.
+  * **Username:** The username with appropriate permissions for accessing the database data you intend to migrate.
+  * **Password:** The password associated with the username.
+
+* Ensure firewalls allow traffic between databases or disable them temporarily.
+* The source Redis database must be SSL-secured, a standard migration requirement.
```

Comment:
This is not true, SSL might be disabled on source service and migration will still work. How about changing this statement to a recommendation? It's best practice to use secure connection everywhere anyway.

Reply:
Update text